src/components/SimpleChart/ChartHeader.tsx

```diff
@@ -0,0 +1,86 @@
+import type { FlexProps } from '@chakra-ui/react'
+import { Flex, Heading } from '@chakra-ui/react'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import type { UTCTimestamp } from 'lightweight-charts'
+import type { ReactElement, ReactNode } from 'react'
+import { useTranslate } from 'react-polyglot'
+import styled from 'styled-components'
+import { Row } from 'components/Row/Row'
+import { useLocaleFormatter } from 'hooks/useLocaleFormatter/useLocaleFormatter'
+
+import { useHeaderDateFormatter } from './hooks'
+
+const ChartHeaderWrapper = (props: FlexProps) => (
```

Comment:
We're not passing any prop in currently, let's remove the props?

Reply:
we are passing children

Reply:
But fair enough don't need to customize this